torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -55,6 +58,104 @@ Value* broadcastSizes(at::ArrayRef<Value*> sizes, AliasDb* db) {
   return broadcast_n->output();
 }
 
+static bool isFwdValue(Value* v, torch::jit::Block* b) {
```

Comment:
nitpick: we should have a place to put shared utility functions for graph mutation.